documentation/man/features/mail.rst

```diff
@@ -20,15 +20,28 @@ It is common to deal with patch submissions to a mailing list, and
 **mail** functionality aims to wrap this tool to simplify its usage and
 integrate it with other **kw** functionalities.
 
+.. warning::
+  If any *<extra-args>* start with a ``-`` they **need** to be placed after the
+  ``--`` argument.
+
 OPTIONS
 =======
 -s, \--send:
   Send an email using ``git send-email`` to the email adresses specified with
-  ``--to``. This sends the last commit.
+  ``--to``. You can provide *<extra-args>* to be passed directly to
+  ``git send-email``, if they start with a ``-`` they **need** to come after
+  the ``--`` argument.
 
 \--to='<recipient>,...':
   Specify the recipients of the email being sent.
 
+<rev-range>...:
+  Specify the *<rev-range>*s to be sent. The last commit is taken as the
```

Comment:
I think this phrase can have two meanings `Specify the *<rev-range>*s to be sent.`:
1. You are sending configurations to git-send-email.
2. You are adding something for sending the patch.
I got what you mean, but let's try to remove ambiguities.